OpenTESArena/src/Interface/CinematicPanel.h

```diff
@@ -22,8 +22,8 @@ class CinematicPanel : public Panel
 	double secondsPerImage, currentSeconds;
 	int imageIndex;
 public:
-	CinematicPanel(Game *game, const std::string &paletteName,
-		const std::string &sequenceName, double secondsPerImage,
+	CinematicPanel(Game *game, const std::string &sequenceName,
+		const std::string &paletteName, double secondsPerImage,
```

Comment:
I didn't notice this bug until now. I wonder why it works. We should match the parameter ordering here with `ImagePanel.h/.cpp` and `ImageSequencePanel.h/.cpp`, where `paletteName` comes before the other strings.